src/odemis/driver/npmc.py

```diff
@@ -46,13 +46,12 @@ class ESPError(model.HwError):
     """
     Exception used to indicate a problem reported by the device.
     """
-
     pass
 
 
 class ESP(model.Actuator):
 
-    def __init__(self, name, role, port, axes=None, inverted=None, **kwargs):
+    def __init__(self, name, role, port, axes=None, inverted=None, position_display_unit="m", **kwargs):
```

Comment:
No need for such "tweaking" argument. The user should use the unit defined in the `.axes`. I'd suggest to not define at all the unit of `.position`.

Reply:
I figured - to avoid confusion, and especially if the conversion factor itself has units (related to converting to "delay seconds" that you should specify what unit the position VA shows externally, so that it is less confusing to understand how it converts to the internal units (which are mm)